src/UserCook/action.js

```diff
@@ -0,0 +1,75 @@
+import axios from "axios";
+export const USER_SIGNED_UP = "user/SIGNED_UP";
+export const USERPROFILE_SIGNED_UP = "userProfile/SIGNED_UP";
+export const USERADDRESS_SIGNED_UP = "userAddress/SIGNED_UP";
+
+const baseUrl = "http://localhost:5000";
```

Comment:
with axios you can create an instance in another file where you set the baseUrl and then import axios from that file from anywhere, try it out!
```
// src/axios.js
import axios from "axios";

const instance = axios.create({
  baseURL: "http://localhost:4000"
});
export default instance;
```

Reply:
thanks! will try it out.

Reply:
got this one to work. very cool suggestion. thanks!